Dynatrace/dynatrace-operator
# Common sanitization logic for all env var and arguments

## Description

Goal of this PR is to introduce a common logic for all env var and argument handling for containers. Basically duplication shall be avoided and arguments/env vars passed by the user always shall take precedence.

## How can this be tested?

Deploy a Dynakube with Active Gate and One Agent, provide custom env var and argument settings and check the daemonset/statefulset.

## Checklist
